manifest.yml

```diff
@@ -9,5 +9,4 @@ applications:
   random-route: true
   services:
   - TodoList-CouchDB
-  buildpack: https://github.com/IBM-Swift/swift-buildpack.git#develop
```

Comment:
We don't need the develop branch for the Buildpack anymore? Have you tried deploying?